.circleci/config.yml

```diff
@@ -15,7 +15,15 @@ workflows:
       - build:
           filters:
             tags:
-              only: /^v.*/
+              only: /^v\d+\.\d+\.\d+-rc.*$/
+            branches:
+              ignore: /.*/
```

Comment:
Is this branches/ignore filter required in this case?

Reply:
I think it is really necessary, so it will trigger only based on the tags. That's right.